jupyter/jupyter-web-app/overlays/aws/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base
```

Comment:
Let's change indent. Most of the yaml in the project has 0 indent for arrays.

Reply:
Fixed.